Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java

```diff
@@ -608,9 +608,9 @@ public boolean isRenderDocumentPdf() {
     public boolean isRenderDocumentText() {
         return getSelectedCurrentDocument().getContentType() != null
             && (getSelectedCurrentDocument().getContentType().equals(GatewayService.CONTENT_TYPE_APPLICATION_XML)
-                || getSelectedCurrentDocument().getContentType().equals(GatewayService.CONTENT_TYPE_TEXT_HTML)
-                || getSelectedCurrentDocument().getContentType().equals(GatewayService.CONTENT_TYPE_TEXT_PLAIN)
-                || getSelectedCurrentDocument().getContentType().equals(GatewayService.CONTENT_TYPE_TEXT_XML));
+            || getSelectedCurrentDocument().getContentType().equals(GatewayService.CONTENT_TYPE_TEXT_HTML)
```

Comment:
Optional: if you are doing an additional commit, you are using getSelectedCurrentDocument().getContentType(). Can you put it to a variable and access that directly?